tests/test_fqdn.py

```diff
@@ -156,3 +156,24 @@ def test_equality_is_case_insensitive(self):
             FQDN("all-letters-were-created-equal.com."),
             FQDN("ALL-LETTERS-WERE-CREATED-EQUAL.COM."),
         )
+
+class TestHash(TestCase):
```

Comment:
Can we add a test for two FQDN objects that should not have the same hash?

Reply:
done